plugins/modules/load_balancer_pool_member.py

```diff
@@ -0,0 +1,258 @@
+uuid:
+  description: The unique identifier for this load balancer pool member
+  returned: success
+  type: str
+  sample: cfde831a-4e87-4a75-960f-89b0148aa2cc
+name:
+  description: The display name of the load balancer pool member
+  returned: success
+  type: str
+  sample: web-lb-pool
+enabled:
+  description: THe status of the load balancer pool member
+  returned: success
+  type: str
+  sample: true
+created_at:
+  description: The creation date and time of the load balancer pool member
+  returned: success when not state == absent
+  type: datetime
+  sample: 2023-02-07T15:32:02.308041Z
+pool:
+  description: The pool of the pool member
+  returned: success
+  type: dict
+  sample: {
+            "href": "https://api.cloudscale.ch/v1/load-balancers/pools/20a7eb11-3e17-4177-b46d-36e13b101d1c",
+            "uuid": "20a7eb11-3e17-4177-b46d-36e13b101d1c",
+            "name": "web-lb-pool"
+            }
+protocol_port:
+  description: The port to which actual traffic is sent
+  returned: success
+  type: str
+  sample: 8080
+monitor_port:
+  description: The port to which health monitor checks are sent
+  returned: success
+  type: str
+  sample: 8081
+address:
+  description: The IP address to which traffic is sent
+  returned: success
+  type: str
+  sample: 10.11.12.3
+subnet:
+  description: The subnet in a private network in which address is located
+  returned: success
+  type: dict
+  sample: {
+            "href": "https://api.cloudscale.ch/v1/subnets/70d282ab-2a01-4abb-ada5-34e56a5a7eee",
+            "uuid": "70d282ab-2a01-4abb-ada5-34e56a5a7eee",
+            "cidr": "10.11.12.0/24"
+            }
+monitor_status:
+  description: The status of the pool's health monitor check for this member
+  returned: success
+  type: str
+  sample: up
+tags:
+  description: Tags assosiated with the load balancer
+  returned: success
+  type: dict
+  sample: { 'project': 'my project' }
+'''
+
+from copy import deepcopy
+
+from ansible.module_utils.basic import AnsibleModule
+from ..module_utils.api import (
+    AnsibleCloudscaleBase,
+    cloudscale_argument_spec,
+)
+
+ALLOWED_STATES = ('present',
+                  'absent',
+                  )
+ALLOWED_ENABLED = ('true',
+                   'false',
+                   )
+
+
+class AnsibleCloudscaleLoadBalancerPoolMember(AnsibleCloudscaleBase):
+
+    def create(self, resource, data=None):
+        super().create(resource, data)
+
+        if not data:
+            data = dict()
+        data = AnsibleCloudscaleLoadBalancerPoolMember.remove_param(self, data)
+
+        if not self._module.check_mode:
+            resource = self.wait_for_state('enabled', ('true', 'false'))
+        return resource, data
+
+    def remove_param(self, data):
+        for param in self.resource_create_param_keys:
+            data[param] = self._module.params.get(param)
+        return data
```

Comment:
What's the purpose of this code? Why can't you just use `AnsibleCloudscaleBase` like in most other load balancer modules?

Reply:
I'll need to call the wait_for_state function and pass non-default variables:
`resource = self.wait_for_state('enabled', ('true', 'false'))`